reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -718,7 +730,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.6.5"
```

Comment:
is there a reason that we downgrade python version?

Reply:
We started the updates when the notebook instances were at Python 3.6.5. The sagemaker team upgraded the notebook environments, so they are now using Python 3.6.10. So the 3.6.5 changes are done before that update.